Mathlib/GroupTheory/SpecificGroups/Coxeter.lean

```diff
@@ -458,7 +457,7 @@ def H₄ : Matrix (Fin 4) (Fin 4) ℕ :=
      2, 2, 5, 1]
 
 theorem H₄IsCoxeter : IsCoxeter H₄ where
-  symmetric := by simp [Matrix.IsSymm]; decide
+  symmetric := by unfold Matrix.IsSymm; decide
```

Comment:
I guess these could have been simp with decide turned on instead? I'm a bit wary of these. Decide is a good example of a tactic that doesn't card too much what the goal is before but might run a lot slower if the input is in a different form